pkgs/development/interpreters/rakudo/moarvm.nix

```diff
@@ -10,6 +10,17 @@ stdenv.mkDerivation rec {
     sha256 = "sha256-fzSHpw6Ld74OTi8SsUxJ9qAdA3jglAyGlYyQFsSVrXU=";
    };
 
+  postPatch = ''
+    patchShebangs .
```

Comment:
Please be more specific which files need patching.

Reply:
Why? There are something like 20 of them. Shouldn't pretty much all scripts be fixed as a matter of course? Missing one has obvious results on linux because of the sandbox, but on macos, largely without that protection, it causes subtle problems.

Reply:
There has been instances in the past where particular large directories have cause quite a bit of time for them to be patched individually.

If it's relatively quick to patch the entire directory, then it's not too much of a concern.

Reply:
Takes about 3 seconds.

Reply:
> Why? There are something like 20 of them. Shouldn't pretty much all scripts be fixed as a matter of course?

patchShebang can also patch directories. Are they all in one or two directories? If so please specify those.